tests/conftest.py

```diff
@@ -204,3 +205,35 @@ def tiny_bert_tokenizer(_session_tiny_bert_tokenizer):
 @pytest.fixture
 def tiny_bert_config(_session_tiny_bert_config):
     return copy.deepcopy(_session_tiny_bert_config)
+
+
+# Note: These methods are an alternative to the tiny_bert fixtures above.
+# Fixtures cannot be used natively as paramaterized inputs, which we require when
+# we wish to run a test across multiple models, one of which is a HuggingFace BERT Tiny.
+# As a workaround, we inject objects into the PyTest namespace. Tests should not directly
+# use pytest.{var}, but instead should import and use the helper copy methods configure_{var}
+# so the objects in the PyTest namespace do not change.
+def pytest_configure():
+    transformers = pytest.importorskip('transformers')
```

Comment:
Is it a problem if the `pytest_configure` call gets skipped (i.e. when running with `dev` instead of `all`)? Probably not? but not sure